eng/pipelines/release-cli.yml

```diff
@@ -986,12 +986,42 @@ stages:
                     filePath: eng/scripts/Set-CliVersionVariable.ps1
                   displayName: Set CLI_VERSION
 
+                - pwsh: |
+                    # Initial upload locations
+                    $publishUploadLocations = 'release/$(CLI_VERSION);release/latest'
+
+                    $isPublishingGa = eng/scripts/Test-ShouldReleasePackageVersion.ps1 `
```

Comment:
In some of the other legs, when we call `eng/scripts/Test-ShouldReleasePackageVersion.ps1` we pass in `-AllowPrerelease` but here we do not.  Should we?

Reply:
I'm guessing not, because if something was pre-release we should not be marking it as stable, but then the other legs do pass this in, so I'm not sure exactly what the expectation is here.

Reply:
The `-AllowPrerelease` variable is now `false` in those invocations (it's a template parameter whose default value is `false`) as it is here.

We only want to execute these code paths on a stable release.